lib/active_merchant/billing/gateways/adyen.rb

```diff
@@ -428,13 +443,14 @@ def cvv_result_from(response)
         CVC_MAPPING[response['additionalData']['cvcResult'][0]] if response.dig('additionalData', 'cvcResult')
       end
 
-      def url
+      def url(action)
+        url_ending = "#{endpoint(action)}#{API_VERSION}"
```

Comment:
Do we need to track different API versions now, each for the `Recurring/` and the `Payment/` endpoint? 

The `Recurring/` endpoint does not seem to have a `v40` that is currently defaulted to for `API_VERSION`.  Don't know how accurate this documentation that I'm referring to is: https://docs.adyen.com/api-explorer/#/Recurring/v30/overview

Reply:
Thanks, Tanya. I had been looking at this page: https://docs.adyen.com/classic-integration/recurring-payments/disable-stored-details

I used that url in communication with Adyen support and sent them examples using `/Recurring/v40/disable`. 

I have sent another email to support asking explicitly about the url you shared and versions supported for `/Recurring`.

Reply:
Ah, okay thanks. The example curls commands they show on the page you shared also use `v25` and `v46` for `Recurring/` and `Payments/`. I wonder if `Recurring/` still accepts `v40` as testing these against their test server has worked?

Reply:
@tanyajajodia I heard back from Adyen:
> V40 is working on test because there is a "back-up" making sure that it is correctly sent to the proper version. I would therefore advice you to use v30 to be able to enjoy all our updates and features improvement on LIVE.

I will update the url-creation to account for this. Great catch!!